wherehows-dao/src/main/java/wherehows/dao/DaoFactory.java

```diff
@@ -24,4 +28,11 @@ public DatasetsDao getDatasetsDao() {
     return datasetsDao;
   }
 
+  public void setEntityManagerFactory(EntityManagerFactory factory) {
```

Comment:
Let's just force the constructor to take factory instead of using getter & setter.